SECP-service/src/main/java/com/visucius/secp/resources/GroupResource.java

```diff
@@ -75,4 +77,17 @@ public Response deletePermissions(@Auth GroupModifyRequest request, @PathParam("
     {
         return  groupController.deletePermissions(request,id);
     }
+
+    @GET
+    @UnitOfWork
+    public Response getAllGroups() {
+        return groupController.getAllGroups();
```

Comment:
Why would a user need to get all the groups? Instead it should get all the groups for a user.